frontend/common/stores/account-store.js

```diff
@@ -186,7 +186,7 @@ const controller = {
           return
         }
 
-        data.setToken(res.key)
+        data.setToken(Project.cookieAuthEnabled?'true' : res.key)
```

Comment:
nit but there's an inconsistency in the formatting of some of these ternary statements.

```javascript
Project.cookieAuthEnabled?'true' : res.key
```

here vs the following in subsequent lines


```javascript
Project.cookieAuthEnabled ? 'true' : data.token
```

Reply:
Fixed.